public/app/features/alerting/unified/components/admin/AlertmanagerConfig.test.tsx

```diff
@@ -132,7 +132,7 @@ describe('Admin config', () => {
     await waitFor(() => expect(mocks.api.updateAlertManagerConfig).toHaveBeenCalled());
     expect(mocks.api.updateAlertManagerConfig.mock.lastCall).toMatchSnapshot();
 
-    await waitFor(() => expect(mocks.api.fetchConfig).toHaveBeenCalledTimes(3));
+    await waitFor(() => expect(mocks.api.fetchConfig).toHaveBeenCalledTimes(2));
```

Comment:
I'm not sure why this was `3`? Reading through the test it should definitely be `2`, once on initial load and once after saving when the `AlertmanagerConfiguration` tag has been invalidated.